### PR TITLE
Bump version to 2.2.0.snapshot2

### PR DESCRIPTION
**Pull request by suyograo**
Bump version, update gem and lock file to point to RubyGems.org
in preperation for 2.2.0 release

### CODE REVIEW

**Comment by colinsurprenant** [comment]
yup, LGTM!

**Comment by colinsurprenant** [comment]
@suyograo checking error, I think it may just be a missing `require` - this is the first time we actually test a "real" package with core-event gem in rubygems.

**Comment by colinsurprenant** [comment]
@suyograo oh sweetjesus :sweat_smile: I finally just found the problem. the core-even gem is malformed, it does not include the correct `/lib` directory... fixing!

**Comment by colinsurprenant** [comment]
actually - both core gems have the root logstash content because they were build from the top level logstash dir. we need to `cd` into each and launch `gem build` in the root of each gem.

the proper fix here is simply to rebuild and push both core gems and update this PR to new gem versions.

**Comment by suyograo** [comment]
Fixed the previous error. Built from within gem dirs, yanked old snapshot1 gems and republished snapshot2 gems

**Comment by colinsurprenant** [comment]
LGTM + Jenkins all green :shipit:

File: Gemfile.jruby-1.9.lock [hunk 1]
```diff
@@ -259,11 +251,12 @@ GEM
       logstash-core (>= 2.0.0.beta2, < 3.0.0)
       nokogiri
       xml-simple
-    logstash-input-beats (2.0.3)
+    logstash-input-beats (2.1.0)
```
Comment (ph) [inline Gemfile.jruby-1.9.lock]: Oh yes!

File: Gemfile.jruby-1.9.lock [hunk 2]
```diff
@@ -430,19 +428,19 @@ GEM
...
       logstash-filter-json
       logstash-output-file
-    logstash-output-elasticsearch (2.3.0-java)
+    logstash-output-elasticsearch (2.4.0-java)
       cabin (~> 0.6)
       concurrent-ruby
       elasticsearch (~> 1.0, >= 1.0.13)
       logstash-core (>= 2.0.0, < 3.0.0)
-      manticore (~> 0.4.2)
+      manticore (>= 0.5.2, < 1.0.0)
```
Comment (ph) [inline Gemfile.jruby-1.9.lock]: Small comment here, this is a pretty open dependency, from what it was before?

**Comment by ph** [comment]
Look good to me!

**Pull request merged**